tox.ini

```diff
@@ -4,6 +4,7 @@ envlist=
     py34-django{18,110,111},
     py35-django{18,110,111},
     py36-django{111},
+    py36-django{20},
```

Comment:
This could be part of the previous line. ie:

```
py36-django{111,20},
```

It's also worth adding `,20` to the `py35` line, as Django 2.0 supports that version of python too.